src/lib/layer.js

```diff
@@ -284,6 +290,16 @@ export default class Layer {
   }
 
   /**
+   * Returns the picking color of the object at 'selectedObjectIndex' prop.
+   * @return {Array} - the picking color or null if 'selectedObjectIndex is invalid.
+   */
+  selectedObjectPickingColor() {
+    if (this.props.selectedObjectIndex < 0) {
+      return null;
```

Comment:
**REQUIRED CHANGE** I don't see apps calling this directly, so this method should be private (start with an underscore and be moved down to the section with private functions).